torch/utils/tensorboard/_convert_np.py

```diff
@@ -32,7 +32,7 @@ def make_np(x):
 
 def _prepare_pytorch(x):
     if isinstance(x, torch.autograd.Variable):
-        x = x.data
+        x = x.detach()
```

Comment:
`Variable` and `Tensor` are the same thing now... The better fix is probably remove `if`, and change next line to `x = x.detach().cpu().numpy()`.